src/spaceObjects/spaceship.cpp

```diff
@@ -116,8 +116,10 @@ SpaceShip::SpaceShip(string multiplayerClassName, float multiplayer_significant_
     impulse_acceleration = 20.0;
     energy_level = 1000;
     max_energy_level = 1000;
+    turnSpeed = 0.0f;
 
     registerMemberReplication(&target_rotation, 1.5);
+    registerMemberReplication(&turnSpeed, 1.5);
```

Comment:
I think 1.5 seconds between turnSpeed updates towards clients will give a few issues with joystick controls looking very jittery on other clients. 0.1 would be a better number. (note that this a minimum update delay, if the turnSpeed is not changed it will not send it at all)

Reply:
addressed. thanks